examples/twittermap/web/public/javascripts/map/services-multilayer-countmap.js

```diff
@@ -0,0 +1,984 @@
+      cloudberry.parameters.geoIds = [];
+      if (polygons !== undefined) {
+        polygons.features.forEach(function (polygon) {
+          if (bounds._southWest.lat <= polygon.properties.centerLat &&
+              polygon.properties.centerLat <= bounds._northEast.lat &&
+              bounds._southWest.lng <= polygon.properties.centerLog &&
+              polygon.properties.centerLog <= bounds._northEast.lng) {
+            cloudberry.parameters.geoIds.push(polygon.properties[idTag]);
+          }
+        });
+      }
+    }
+
+    function zoomFunction(instance){
+      var scope = instance.scope;
+      function resetGeoInfo(level) {
+        cloudberry.parameters.geoLevel = level;
+        cloudberry.parameters.geoLevel = level;
+        if (instance.geojsonData[level]){
+          resetGeoIds(instance.bounds, instance.geojsonData[level], level + "ID");
+        }
+      }
+      if (instance.map) {
+        instance.status.zoomLevel = instance.map.getZoom();
+        instance.bounds = instance.map.getBounds();
+        if (instance.status.zoomLevel > 9) {
+          resetGeoInfo("city");
+          if (instance.polygons.statePolygons) {
+            instance.layer.removeLayer(instance.polygons.statePolygons);
+          }
+          if (instance.polygons.countyPolygons) {
+            instance.layer.removeLayer(instance.polygons.countyPolygons);
+          }
+          if (instance.polygons.stateUpperPolygons) {
+            instance.layer.removeLayer(instance.polygons.stateUpperPolygons);
+          }
+          instance.layer.addLayer(instance.polygons.countyUpperPolygons);
+          loadCityJsonByBound(instance.onEachFeature,instance);
+        } else if (instance.status.zoomLevel > 5) {
+          resetGeoInfo("county");
+          scope.resetGeoInfo("county");
+          //TODO publish events
+          //moduleManager.publishEvent(moduleManager.EVENT.CHANGE_ZOOM_LEVEL, {level: instance.map.getZoom(), bounds: instance.map.getBounds()});
+          if (instance.polygons.statePolygons) {
+            instance.layer.removeLayer(instance.polygons.statePolygons);
+          }
+          if (instance.polygons.cityPolygons) {
+            instance.layer.removeLayer(instance.polygons.cityPolygons);
+          }
+          if (instance.polygons.countyUpperPolygons) {
+            instance.layer.removeLayer(instance.polygons.countyUpperPolygons);
+          }
+          instance.layer.addLayer(instance.polygons.stateUpperPolygons);
+          instance.layer.addLayer(instance.polygons.countyPolygons);
+        } else if (instance.status.zoomLevel <= 5) {
+          resetGeoInfo("state");
+          if(!instance.status.init){
+            //TODO publish events
+            //moduleManager.publishEvent(moduleManager.EVENT.CHANGE_ZOOM_LEVEL, {level: instance.map.getZoom(), bounds: instance.map.getBounds()});
+          }
+          if (!instance.status.init) {
+          cloudberry.query(cloudberry.parameters);
```

Comment:
This is from old code, not even from current master, let's remove it. I think you need a better development environment.